build/sortObject.mjs

```diff
@@ -1,16 +1,16 @@
-'use strict';
-
 /**
  * Sorting objects by keys
  * https://www.w3docs.com/snippets/javascript/how-to-sort-javascript-object-by-key.html
  * @param {Record<string, *>} obj object to be sorted
  * @returns {Record<string, *>} same as {@param obj} but sorted keys
  */
-module.exports = (obj) => {
+export function sortObject(obj) {
   return Object.keys(obj)
     .sort()
     .reduce(function (result, key) {
       result[key] = obj[key];
       return result;
     }, {});
-};
+}
```

Comment:
why switch from closure to function?
just name the single closure as a const and export it

Reply:
sure i just think function is a more concise way to declare a function when there's no need for the closured feature

Reply:
yeah, but our styling, i'm pretty sure, specifies avoiding functions entirely